src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -8,20 +16,14 @@
 {# Labels #}
 
 {% block form_label -%}
-{% spaceless %}
     {% if label is same as(false) %}
         <div class="{{ block('form_label_class') }}"></div>
     {% else %}
```

Comment:
these tags are missing the whitespace control to remove indentation spaces

Reply:
changed. please review the recent commit, thanks!